system/tests/ko7/I18nTest.php

```diff
@@ -9,9 +8,10 @@
  *
  * @package    KO7
  * @category   Tests
- * @author     Kohana Team
- * @author     Jeremy Bush <contractfrombelow@gmail.com>
- * @copyright  (c) Kohana Team
+ *
+ * @author     Kohana Team, Jeremy Bush <contractfrombelow@gmail.com>
+ * @copyright  (c) 2008´- 2016 Kohana Team
```

Comment:
2008´? Think it is 2008 - 2016 or did i get something wrong? :)

Reply:
Yes ofc ;) thanks for the details 😄